samples/client/petstore/typescript-angular-v2/with-interfaces/api/user.serviceInterface.ts

```diff
@@ -11,8 +11,7 @@
  */
 import { Headers }                                           from '@angular/http';
 
-import { Observable }                                        from 'rxjs/Observable';
-
+import { Observable }                                    from 'rxjs/Observable';
```

Comment:
did you re-generate the v2 sample files? this line should not change...

Reply:
I re-generated by using ```./bin/typescript-angular-petstore-all.sh ```.  The padding was different in the apiInterface.mustache, I just committed a fix.